exchequer_client.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exchequer_client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "exchequer_client"
+  spec.version       = ExchequerClient::VERSION
+  spec.authors       = ["DailyDrip"]
+  spec.email         = ["devs@dailydrip.com.com"]
+
+  spec.summary       = %q{TODO: Write a short summary, because Rubygems requires one.}
+  spec.description   = %q{TODO: Write a longer description or delete this line.}
+  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
+  spec.license       = "MIT"
+
```

Comment:
lets fill in this section,  maybe something like 'Rails Client for the Exchequer payments system' and link to the server github page for the homepage?